rts/lra/service-base/src/main/java/io/narayana/lra/logging/LRALogger.java

```diff
@@ -29,5 +29,5 @@ private LRALogger() {
     }
 
     public static final Logger logger = Logger.getLogger("io.narayana.lra");
-    public static final lraI18NLogger i18NLogger = Logger.getMessageLogger(lraI18NLogger.class, "io.narayana.lra");
+    public static final LraI18nLogger i18nLogger = Logger.getMessageLogger(LraI18nLogger.class, "io.narayana.lra");
```

Comment:
The naming should be (probably) left as it was, i.e. `lraI18NLogger` and `i18NLogger` - it's because of consistency. Narayana project just uses this naming convention for `I18n` classses - see https://github.com/jbosstm/narayana/blob/master/ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/logging/arjunaI18NLogger.java, https://github.com/jbosstm/narayana/blob/master/ArjunaJTA/jta/classes/com/arjuna/ats/jta/logging/jtaI18NLogger.java or whatever else.

Reply:
ok, I didn't know that this is customary. But still why is it a Java class starting with lowercase letter and if we use Java convention of PascalCase or camelCase then only new words in the name should be capitalized, so i18*N*Logger is wrong because N is not a new word in i18n. But I understand that it is done this way, I will change it back, but this is not a correct Java form.

Reply:
Agree, it's not a correct form from Java checkstyle guidelines. Here I was thinking that as the LRA is part of the Narayana project it would be good to follow the style that Narayana uses for all other i18n classes. I feel the project consistency will be be better in this case.